src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -0,0 +1,53 @@
+@echo off 
+setlocal
+
+REM 
+REM Start WCF Self Hosted Service on local machine
+REM
+
+REM Build tools
+call BuildWCFSelfHostedService.cmd
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while building WCF Self hosted Service.
+    set __EXITCODE=%ERRORLEVEL% 
+    goto :done
+  )
+call BuildCertUtil.cmd
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while building the Certificate generator.
+    set __EXITCODE=%ERRORLEVEL%
+    goto :done
+  )
+
+
+REM Config Certs
+REM we need the direcotry to save the test.crl file. We are investigate a way to get rid of it
+REM
+md c:\wcftest
+pushd ..\..\..\..\bin\Wcf\tools\CertificateGenerator
+call CertificateGenerator.exe
+set __EXITCODE=%ERRORLEVEL%
+popd
+if NOT [%__EXITCODE%]==[0] (
+    echo Warning: An error occurred while running certificate generator.
+  )
```

Comment:
Great area for later discussion regarding [ConditionalFact].  If we can't install firewall rules. it would be great for those tests to be skipped with "Condition(s) not met: firewall rule for port NNN installed".  This could involve setup scripts leaving "breadcrumb" files that [ConditionalFact] can access.  Definitely not needed for this PR!  I do agree in principal that if certain parts of setup fail, we should not bail out entirely, but should leave a clear log of what went wrong and potentially run the tests that can run.